dist/array_extensions.js

```diff
@@ -160,5 +160,20 @@
 
 		let cb = isFunction ? (a, b) => a + spec(b) : (a, b) => a + b;
 		return this.reduce(cb, 0);
+    }
+
+    function flatten(container=null) {
+        var result = container ? container : []; 
```

Comment:
A very useful sintaxis to accomplish the same result is `var result = container || [];`